src/pages/Search/SearchAdvancedFiltersPage/SearchFiltersParticipantsSelector.tsx

```diff
@@ -0,0 +1,172 @@
+import React, {useCallback, useEffect, useMemo, useState} from 'react';
+import {useOnyx} from 'react-native-onyx';
+import Button from '@components/Button';
+import {usePersonalDetails} from '@components/OnyxProvider';
+import {useOptionsList} from '@components/OptionListContextProvider';
+import SelectionList from '@components/SelectionList';
+import InviteMemberListItem from '@components/SelectionList/InviteMemberListItem';
+import useLocalize from '@hooks/useLocalize';
+import * as DeviceCapabilities from '@libs/DeviceCapabilities';
+import * as OptionsListUtils from '@libs/OptionsListUtils';
+import type {Option} from '@libs/OptionsListUtils';
+import type {OptionData} from '@libs/ReportUtils';
+import Navigation from '@navigation/Navigation';
+import CONST from '@src/CONST';
+import ONYXKEYS from '@src/ONYXKEYS';
+import ROUTES from '@src/ROUTES';
+
+const defaultListOptions = {
+    userToInvite: null,
+    recentReports: [],
+    personalDetails: [],
+    currentUserOption: null,
+    headerMessage: '',
+    categoryOptions: [],
+    tagOptions: [],
+    taxRatesOptions: [],
+};
+
+function getSelectedOptionData(option: Option): OptionData {
+    return {...option, selected: true, reportID: option.reportID ?? '-1'};
+}
+
+type SearchFiltersParticipantsSelectorProps = {
+    initialAccountIDs: string[];
+    onFiltersUpdate: (accountIDs: string[]) => void;
+};
+
+function SearchFiltersParticipantsSelector({initialAccountIDs, onFiltersUpdate}: SearchFiltersParticipantsSelectorProps) {
+    const {translate} = useLocalize();
+    const personalDetails = usePersonalDetails();
+    const {options, areOptionsInitialized} = useOptionsList();
+
+    const [betas] = useOnyx(ONYXKEYS.BETAS);
+    const [selectedOptions, setSelectedOptions] = useState<OptionData[]>([]);
+    const [searchTerm, setSearchTerm] = useState('');
+    const cleanSearchTerm = useMemo(() => searchTerm.trim().toLowerCase(), [searchTerm]);
+
+    const defaultOptions = useMemo(() => {
+        if (!areOptionsInitialized) {
+            return defaultListOptions;
+        }
+
+        return OptionsListUtils.getFilteredOptions(options.reports, options.personalDetails, betas, '', selectedOptions);
```

Comment:
I think we should copy the other options from [MoneyRequestParticipantsSelector](https://github.com/Expensify/App/blob/65bca9172acebe12236f72296f94b8b93f53a61d/src/pages/iou/request/MoneyRequestParticipantsSelector.tsx#L98-L128), most importantly:

1. Exclude expensify emails, e.g. concierge
2. Include owned Workspace chats
3. Include P2P
4. Include self DM

```suggestion
        return OptionsListUtils.getFilteredOptions(
            options.reports,
            options.personalDetails,
            betas,
            '',
            selectedOptions,
            CONST.EXPENSIFY_EMAILS,
            true,
            true,
            false,
            {},
            [],
            false,
            {},
            [],
            true,
            false,
            false,
            CONST.IOU.MAX_RECENT_REPORTS_TO_SHOW,
            {},
            true,
            false,
            [],
            true,
        );
```